chompfile.toml

```diff
@@ -0,0 +1,78 @@
+version = 0.1
+
+[[task]]
+name = 'build'
+run = 'tsup src/loader.ts --format esm,cjs --dts'
```

Comment:
We can add `deps` and add `npm:install`. So, if we run `chomp` by default. The build will make sure the deps are also installed 👍